packages/astro-embed-baseline-status/BrowserIcon.astro

```diff
@@ -0,0 +1,16 @@
+---
+import type { Browser } from './types';
+
+interface Props {
+	browser: Browser;
+}
+
+const ICONS = {
+	chrome: `<svg xmlns="http://www.w3.org/2000/svg" width="21" height="21" fill="none"><g clip-path="url(#a)"><path fill="#fff" d="M10.2 15.3a5.1 5.1 0 1 0 0-10.2 5.1 5.1 0 0 0 0 10.2Z"/><path fill="#229342" d="M5.783 12.75 1.367 5.103A10.198 10.198 0 0 0 10.201 20.4l4.416-7.65a5.098 5.098 0 0 1-8.834.001Z"/><path fill="#FBC116" d="M14.617 12.75 10.2 20.4a10.199 10.199 0 0 0 8.831-15.3H10.2a5.1 5.1 0 0 1 4.418 7.65Z"/><path fill="#229342" d="M10.2 14.238a4.038 4.038 0 1 0 0-8.075 4.038 4.038 0 0 0 0 8.075Z"/><path fill="#E33B2E" d="M10.2 5.1h8.833a10.198 10.198 0 0 0-17.666.002l4.416 7.649h.002a5.1 5.1 0 0 1 4.415-7.65Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20.4v20.4H0z"/></clipPath></defs></svg>`,
```

Comment:
These icons set things like `id="a"` and `id="b"`, which pollute the global namespace.

This is one of the reasons I'm using shadow DOM in my implementation.

Reply:
All of the icons could probably be optimized through SVGO as well.

Reply:
Ran the browser icons through SVGO (most were kind of OK except for the Safari icon which was really bloated).

I also moved them to be external SVGs rendered via `<img>`:
- avoids repeating HTML markup & supports caching
- avoids namespace issues with `id`s etc.
- seems safe as these don’t really need direct styling via CSS variables etc.